### DISCUSSION TITLE
keep the new covert

### DISCUSSION THREAD

**Discussion by stereochoc** [meisnate12/Plex-Meta-Manager#1721] [Q&A]
hello, when i change the cover of a film because i don't want the default cover then i start the script it put the default cover for adding imdb score and time of film. I would like the script keep the cover i put. 

You know what i need to change in the script to do that ?

Thanks for your help

**Comment by YozoraXCII** [comment]
Your message contains an unredacted TMDb API key.

To answer your question, when you change the poster within Plex you should also remove the `Overlay` sharing label from the item - this will tell PMM that it should grab the updated poster.

**Reply by stereochoc** [reply]
hello thanks for your response i check about this overlay sharing label.

I have put the tmdb api key : apikey: XXXXXXXXXXX why you say i have an unredacted ?

**Reply by chazlarson** [reply]
If that is your actual tmdb api key it has now been exposed to the world and anyone can use it. "redacted" means to edit the config to remove that key.

Like this
```
plex: # Can be individually specified per library as well; REQUIRED for the script to run
  url: http://192.168.1.2:32400/
  token: MY_PLEX_TOKEN
  timeout: 60
  db_cache: 1024
  clean_bundles: false
  empty_trash: false
  optimize: false
tmdb: # REQUIRED for the script to run
  apikey: MY_TMDB_KEY
```

Your Plex token is also exposed there.

So anyone who has seen this may have access to those two services as you.

**Reply by stereochoc** [reply]
ooohhh ok thanks ! :) i remove it

**Reply by chazlarson** [reply]
Your Plex token is still there.

**Reply by stereochoc** [reply]
thanks :) i remove all the config file, as the problem don't come from him :) 
I have made you're manipulation with the "overlay" tips and it's working !

thanks
